src/packer.pkr.hcl

```diff
@@ -80,9 +80,8 @@ locals { timestamp = regex_replace(timestamp(), "[- TZ:]", "") }
 source "amazon-ebs" "windows" {
   ami_block_device_mappings {
     delete_on_termination = true
-    device_name           = "/dev/xvda"
-    encrypted             = true
-    volume_size           = 8
+    device_name           = "/dev/sda1"
+    volume_size           = 1000
```

Comment:
Does this really need to be a terabyte? Why did you remove encryption?

Reply:
Addressed above